2021-22/2021-12-06_70p

```diff
@@ -3,61 +3,58 @@
 from vexcode_vrc import *
 from math import sqrt
 
+def main():
+    drivetrain.set_drive_velocity(100,PERCENT)
+    fork_motor_group.spin_to_position(1800, DEGREES, wait=False)
+    goto( -920,  920, 0)
+    goto( -920,-1450, 0)
+    pick_up()
+    goto(  750,-1200, 0)
+    set_down()
+    goto( 1000,-1300, 1)
+    goto(  920, 1500, 0)
+    pick_up()
+    goto( -600, 1320, 0)
+    set_down()
+
+    goto(  600, 1320, 1)
+    goto(  600, -600, 1)
+    goto( 1400,-1350, 1)
+    goto( 1500, -900, 0)
+    pick_up()
+    goto( 1500,  100, 0)
+    set_down()
+    stop_project()
+
 def goto(target_x, target_y, reverse):
     x1 = gps.x_position(MM)
     y1 = gps.y_position(MM)
-    x2 = target_x
-    y2 = target_y
-    head = gps.heading()   # north is 0, but for the robot 0 is east - from start
-    delta_x = x2 - x1
-    delta_y = y2 - y1
+    delta_x = target_x - x1
+    delta_y = target_y - y1
     distance = math.sqrt(delta_x**2 + delta_y**2)     # pythagorean theorem
-    if ( delta_x ):
-        directio = - math.atan(delta_y / delta_x) * 180 / math.pi
+    if ( delta_x == 0 ):
+        if ( delta_y < 0):
+            direction = 90
+        else:
+            direction = 270
```

Comment:
Finally fixed, not only moving down if delta_x is zero

Reply:
Great, thanks! Now the GOTO function seems complete